### PR TITLE
29 39 preprocessing

### PR DESCRIPTION
**Pull request by yiwen-h**
Have fixed issues 29 and 39.
Completely separated text preprocessing (feature engineering to create sentiment scores and text_length, some cleaning e.g. removing '\n', '\r', lowercasing anything not in allcaps, from the pipeline.
From now data fed to pipeline should first be prepared using factory_data_load_and_split.process_data function (might move it somewhere else, just have one module containing all the preprocessing functions in one place, currently they are scattered).

### CODE REVIEW

File: execution/execution_label.py [hunk 1]
```diff
@@ -4,20 +4,20 @@
     text_classification_pipeline(filename='datasets/text_data.csv', target="label", predictor="feedback",
                                  test_size=0.33,
                                  ordinal=False,
-                                 tknz="spacy",
+                                 tknz="wordnet",
```
Comment (ChrisBeeley) [inline execution/execution_label.py:7]: You're switching from spacy to wordnet, is that right? Can you say why?

The licence is a bit weird, I think it's fine but worth knowing https://wordnet.princeton.edu/license-and-commercial-use

Comment (yiwen-h) [reply execution/execution_label.py:7]: I was just testing it out, it was already an option in the code (https://github.com/CDU-data-science-team/pxtextmining/blob/development/pxtextmining/helpers/tokenization.py) and I wanted to see how much of a difference it made (and also if it still worked with all the preprocessing changes I made). can switch it back.

Comment (ChrisBeeley) [reply execution/execution_label.py:7]: Oh, I see. Well, let's see if it makes a difference then. Just curious, I didn't know it was in there before

**Review by ChrisBeeley — COMMENTED** [review]
:+1:

**Pull request merged**